utils/path_management.py

```diff
@@ -57,4 +57,6 @@ def get_datasets(config):
         if no_sub:
             dataset_name = f'{dataset["data"]}'
             datasets[dataset_name] = Path(data_dir, dataset_name)
+        if 'train_data_name' in config:
+        datasets['train-clean-360'] = Path(data_dir, 'train-clean-360')
```

Comment:
This function is used in two places:
1) https://github.com/Voice-Privacy-Challenge/Voice-Privacy-Challenge-2024/blob/1e9e29865b45ecafbc4f31aaafdb46c5fe5e6325/run_anonymization.py#L16
2) https://github.com/Voice-Privacy-Challenge/Voice-Privacy-Challenge-2024/blob/fix_bugs_v0/utils/prepare_results_in_kaldi_format.py#L46
These two lines are only affected when using configs/eval_anon.yaml (which has the "train_data_name" parameter).  Required to add that "train-clean-360$suffix_anon" has proper Kaldi-style meta files in 2).

Reply:
For the first one, we pass the data to anonymize here in the datasets entry:
https://github.com/Voice-Privacy-Challenge/Voice-Privacy-Challenge-2024/blob/1e9e29865b45ecafbc4f31aaafdb46c5fe5e6325/configs/anon_mcadams.yaml#L20
The train dataset is treated as a traditional dataset. (config utt/spk level with
```
  anon_level_spk: [IEMOCAP_test, IEMOCAP_dev, libri_dev, libri_test]
  anon_level_utt: [train-clean-360]
  ```
). for the check_kaldi_formart_data, it will be executed by the run_anonymization.py part. So it is not necessary either.